bundles/org.openhab.binding.pentair/README.md

```diff
@@ -208,6 +208,8 @@ sitemap pool label="Pool stuff" {
 
 Setting up RS485 and basic protocol - https://www.sdyoung.com/home/decoding-the-pentair-easytouch-rs-485-protocol/
 ser2sock GitHub - https://github.com/nutechsoftware/ser2sock
+Alternative Linux daemon to control Aqualink RS pool controllers.
+Provides web UI, MQTT client & HTTP API endpoints - https://github.com/sfeakes/AqualinkD
```

Comment:
There needs to be an empty line. Otherwise the whole section will be renderered in one line.

Beside that, I really don't understand the addendum. What's the relation to openHAB?

@jsjames can you take a look?

Reply:
I think those two lines were meant as one item in the list of references. I don't really see the point in adding it here, though.

Reply:
ok updated path

Reply:
Thanks for the update! Can you say a few words why this should be added to the readme?

Reply:
Couple reasons.  AqualinkD has documentation on RS-485 and RS-485 debugging.  It supports a variety of different pool equipment from Jandy/Zodiac.

Jandy Aqualink 6524 REV GG | Everything working
Jandy AquaLinkRS 8157 REV JJ | Everything working
Jandy AquaLinkRS 8157 REV MMM | Everything working
Jandy AquaLinkRS 8159 REV MMM | Everything working
Jandy AquaLinkRS B0029221 REV T | Everything working
Jandy AquaLinkRS B0029223 REV T.2 | Everything working
Jandy AquaLinkRS B0029235 REV T.1 | Everything working
Jandy iAqualink E0260801 REV R | Everything working
AquaLink PDA / AquaPalm | Works, please see WiKi for limitations

Reply:
AqualinkD -> mosquitto -> openhab

There are different types of pool controllers.  If you have an AquaLink PDA this Pentair plugin will not work.  An alternative that will work is to use AqualinkD.  Trying to help people find this alternate solution.

Reply:
@ballle98 Ok, now I understand. The correct place would be to post the solution here: https://community.openhab.org/c/tutorials-examples/solutions/50 The readme you edited belongs to a binding, i.e. another solution, so this will be kept closed.